malasakit-django/pcari/apps.py

```diff
@@ -1,11 +1,12 @@
 from __future__ import unicode_literals
+
 import logging
 import os
 import sys
+import yaml
 
 from django.apps import AppConfig
 from django.conf import settings
-import yaml
```

Comment:
PyYAML is not a standard library (which I also forget sometimes), so it should not be in [the first group of imports](https://www.python.org/dev/peps/pep-0008/#imports).